File: backend/src/main/java/by/kapitonov/cinema/backend/service/dto/UpdateFilmSessionDTO.java
package by.kapitonov.cinema.backend.service.dto;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode
public class UpdateFilmSessionDTO {

    private String filmName;
    private Integer ticketCost;
    private Long hallId;
    private Long managerId;
    private String status;
    private Long filmId;

}

File: frontend/src/app/model/dto/film-session/UpdateFilmSessionDTO.ts
import {FilmSession} from "../../FilmSession";

export class UpdateFilmSessionDTO {

  filmName?: string;
  ticketCost?: number;
  hallId?: number;
  managerId?: number;
  status?: string;
  filmId?: number;
  showTime?: string;


  constructor(filmSession: FilmSession) {
    this.filmName = filmSession.filmName;
    this.ticketCost = filmSession.ticketCost;
    this.status = filmSession.status;
    this.showTime = filmSession.showTime;
  }
}
